Woof/Woof/Model/Preferences/PreferencesHandler.swift

```diff
@@ -0,0 +1,57 @@
+import Foundation
+
+/**
+ A type used for managing user preferences in an application.
+ */
+enum PreferencesHandler {
+    /**
+     Saves the selected role as a user preference.
+
+     - Parameters:
+        - selectedRole: The selected role to be saved.
+
+     - Returns: A boolean value indicating whether the saving operation was successful.
+     */
+    @discardableResult static func set(userRole: Role) -> Bool {
+        var preferences: Preferences = loadPreferences() ?? Preferences()
+
+        preferences.selectedRole = userRole
+
+        return store(preferences)
+    }
+
+    /**
+     Loads the selected role from user preferences.
+
+     - Returns: The selected role for the user in the app session. If the role couldn't be loaded, returns `.none`.
+     */
+    static func getRole() -> Role {
```

Comment:
If you set "userRole" somewhere, then be consistent and get "userRole":
```suggestion
    static func getUserRole() -> Role {
```

Reply:
fixed